CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
+    {
+        // GET api/<controller>
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/<controller>/5
+        public string Get(int id)
+        {
+            return "value1";
+        }
+
+        // POST api/<controller>
+        public void Post([FromBody]string value)
```

Comment:
This is one way of doing it, however I think we need an extra level of detail here. TestController implies that our endpoint is /Test, whereas it is more likely to be /Test/{Something} - something to consider for the future?